### PR TITLE
magento/devdocs#7364: Update the storeConfig query to include the new attributes

### PR DESCRIPTION
**Pull request by atwixfirster**
## Purpose of this pull request

This pull request (PR) updates `storeConfig` query with customer configuration settings.

The customer attributes were added in https://github.com/magento/magento2/pull/27876

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.4/graphql/queries/store-config.html

## Links to Magento source code

-  [base store settings](https://github.com/magento/magento2/blob/2.4-develop/app/code/Magento/StoreGraphQl/Model/Resolver/Store/StoreConfigDataProvider.php#L71)
-  [extended store settings](https://github.com/magento/magento2/blob/2.4-develop/app/code/Magento/StoreGraphQl/Model/Resolver/Store/StoreConfigDataProvider.php#L102)
- [declaration in di.xml](https://github.com/magento/magento2/blob/2.4-develop/app/code/Magento/CustomerGraphQl/etc/graphql/di.xml#L26)
- [declaration in schema.graphqls](https://github.com/magento/magento2/blob/2.4-develop/app/code/Magento/CustomerGraphQl/etc/schema.graphqls#L4)

whatsnew
Updated the GraphQL [`storeConfig` query](https://devdocs.magento.com/guides/v2.4/graphql/queries/store-config.html) to include new customer configuration settings.

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/guides/v2.4/graphql/queries/store-config.md [hunk 1]
```diff
@@ -185,6 +175,36 @@ The following query returns information about the store's catalog configuration.
 }
 ```
 
+### Query a store's Customer configuration
```
Comment (rogyar) [inline src/guides/v2.4/graphql/queries/store-config.md:178]: Just to keep the style the same for all titles 

```suggestion
### Query a store's customer configuration
```

File: src/guides/v2.4/graphql/queries/store-config.md [hunk 2]
```diff
@@ -301,6 +321,16 @@ Attribute |  Data Type | Description | Example
 `root_category_id` | Int | The ID of the root category
 `title_separator` | String | Identifies the character that separates the category name and subcategory in the browser title bar | `-`
 
+### Supported Catalog attributes
```
Comment (rogyar) [inline src/guides/v2.4/graphql/queries/store-config.md:324]:
```suggestion
### Supported catalog attributes
```

File: src/guides/v2.4/graphql/queries/store-config.md [hunk 3]
```diff
@@ -185,6 +175,36 @@ The following query returns information about the store's catalog configuration.
...
+**Request:**
+
+```graphql
+{
+    storeConfig {
+        autocomplete_on_storefront,
+        minimum_password_length,
+        required_character_classes_number,
+    }
+}
```
Comment (rogyar) [inline src/guides/v2.4/graphql/queries/store-config.md:191]: Let's save some chars 😎

```suggestion
    storeConfig {
        autocomplete_on_storefront
        minimum_password_length
        required_character_classes_number
    }
}
```

File: src/guides/v2.4/graphql/queries/store-config.md [hunk 4]
```diff
@@ -301,6 +321,16 @@ Attribute |  Data Type | Description | Example
 `root_category_id` | Int | The ID of the root category
 `title_separator` | String | Identifies the character that separates the category name and subcategory in the browser title bar | `-`
 
+### Supported Catalog attributes
+
+Use the `customer` attributes to retrieve information about the store's customer settings. These attributes are defined in the `CustomerGraphQl` module.
+
+Attribute |  Data Type | Description | Configuration path | Example
```
Comment (rogyar) [inline src/guides/v2.4/graphql/queries/store-config.md:328]: Please, rename the "Example" column to "Default Value" or so, to reflect the actual purpose of the column.

File: src/guides/v2.4/graphql/queries/store-config.md (hunk 2 above)
Comment (rogyar) [reply src/guides/v2.4/graphql/queries/store-config.md:324]: By the way, it's rather about the customer attributes than the catalog attributes

File: src/guides/v2.4/graphql/queries/store-config.md (hunk 4 above)
Comment (atwixfirster) [reply src/guides/v2.4/graphql/queries/store-config.md]: **done**

File: src/guides/v2.4/graphql/queries/store-config.md (hunk 2 above)
Comment (atwixfirster) [reply src/guides/v2.4/graphql/queries/store-config.md]: **fixed**

File: src/guides/v2.4/graphql/queries/store-config.md (hunk 3 above)
Comment (atwixfirster) [reply src/guides/v2.4/graphql/queries/store-config.md]: Agreed :)

**fixed**

File: src/guides/v2.4/graphql/queries/store-config.md (hunk 1 above)
Comment (atwixfirster) [reply src/guides/v2.4/graphql/queries/store-config.md]: **fixed**

**Comment by atwixfirster** [comment]
@rogyar , all your suggestions have been implemented!

Could you please verify and bless this PR?

Thank you!

**Comment by keharper** [comment]
running tests

**Pull request merged**